context-monitoring/src/main/java/de/atb/context/monitoring/config/models/datasources/MessageBrokerDataSourceOptions.java

```diff
@@ -0,0 +1,69 @@
+package de.atb.context.monitoring.config.models.datasources;
+
+/*
+ * #%L
+ * ATB Context Monitoring Core Services
+ * %%
+ * Copyright (C) 2015 - 2021 ATB
+ * %%
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * #L%
+ */
+
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * WebServiceDataSourceOptions
+ * 
+ * @author scholze
+ * @version $LastChangedRevision: 143 $
+ * 
+ */
+public enum MessageBrokerDataSourceOptions implements IDataSourceOptionValue {
+
+	PollingInterval("interval", Long.class),
```

Comment:
Do we need this? Usually, the broker pushes new messages to the subscribers and there is no polling involved

Reply:
yes it is not needed here